### PR TITLE
create extension popup UI

### LINKED ISSUE #92: Popup - create UI components as seen on FIGMA
- 1 common component for each network (configuration file maybe?)
- 1 component for each open tab
Screenshot 2021-02-09 at 5 02 00 PM

### PR DESCRIPTION
**Pull request by wirednkod**
Resolve #92

### CODE REVIEW

**Comment by raoulmillais** [comment]
I get this error in the devtools for the popup:

```
popup.200b4782.js:433 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `Popup`. See https://reactjs.org/link/warning-keys for more information.
    at PolkaFont (chrome-extension://mclfgbjfjilokfafahobaaaidancicle/popup.200b4782.js:85156:19)
    at Popup (chrome-extension://mclfgbjfjilokfafahobaaaidancicle/popup.200b4782.js:85564:17)
 ```

**Comment by raoulmillais** [comment]
There's a warning in firefox and various style-related problems:

**Comment by raoulmillais** [comment]
I get this error in the firefox dev-tools:

```
TypeError: can't access dead object4 webconsole.js:268:9
    _getWindowForBrowserConsole webconsole.js:268
    get global webconsole.js:252
    makeDebuggeeValue webconsole.js:492
    arguments webconsole.js:2026
    map self-hosted:221
    prepareConsoleMessageForRemote webconsole.js:2025
    onConsoleAPICall webconsole.js:1750
    onConsoleAPICall self-hosted:1176
    observe console-api.js:94
    CS_recordEvent ConsoleAPIStorage.jsm:123
    Pe Keyframes.js:20
    Ye StyledComponent.js:265
    s StyledComponent.js:265
    PolkaFont PolkaFont.tsx:9
    React 14
    parcelRequire<["popup.tsx"]< popup.tsx:5
    newRequire popup.200b4782.js:47
    parcelRequire popup.200b4782.js:81
    <anonymous> popup.200b4782.js:120
 ```

File: projects/smoldot-extension/src/components/AntSwitch.tsx
Comment (raoulmillais) [inline projects/smoldot-extension/src/components/AntSwitch.tsx:16]: Why is this called "AntSwitch" out of interest?  Is this a special term?

**Review by raoulmillais — CHANGES_REQUESTED** [review]
I'll defer to @Stefie for a proper review of the react code, but it looks good to me.  There are a few errors and issues running it in FF and one error in Chrome that I think you should fix before we merge this.

File: projects/smoldot-extension/src/components/AntSwitch.tsx
Comment (wirednkod) [reply projects/smoldot-extension/src/components/AntSwitch.tsx:16]: I think they gave from material UI this name cause it's design is very similar to AntDesign. I kept it as of tutorials.

**Review by Stefie — CHANGES_REQUESTED** [review]
Feel free to merge. 
I would rename the project to something other than `smoldot-extension` to not start the next batch of confusion, though.
It's a `substrate-connect` extension, smoldot delivers the Wasm Light Client, but the extension should behave the same no matter if the Light client is generated by smoldot or maybe Substrate or Polkadot one day.
The `smoldot-provider` makes 100% sense since that's a wrapper especially for smoldot, bur for all demo apps it doesn't matter where the Light Client is coming from, therefore I'd not name applications after it.

**Comment by raoulmillais** [comment]
> Feel free to merge.
> I would rename the project to something other than `smoldot-extension` to not start the next batch of confusion, though.
> It's a `substrate-connect` extension, smoldot delivers the Wasm Light Client, but the extension should behave the same no matter if the Light client is generated by smoldot or maybe Substrate or Polkadot one day.
> The `smoldot-provider` makes 100% sense since that's a wrapper especially for smoldot, bur for all demo apps it doesn't matter where the Light Client is coming from, therefore I'd not name applications after it.

Yeah I was thinking this as well after our conversation yesterday :+1:  I don't think it should block this PR - I'll open an issue.

**Review by Stefie — APPROVED** [review]

**Pull request merged**
